src/liblibc/lib.rs

```diff
@@ -4632,7 +4646,7 @@ pub mod consts {
             pub const F_GETLK : c_int = 7;
             pub const F_SETLK : c_int = 8;
             pub const F_SETLKW : c_int = 9;
-            pub const F_DUPFD_CLOEXEC : c_int = 10;
+            pub use consts::os::posix08::F_DUPFD_CLOEXEC;
```

Comment:
Actually, #27930 already introduced a breaking change to `libc`, which changed the location of `_PC_NAME_MAX` from the `funcs` module (!) to the `consts` module. So I think it would be clearer if this got removed completely, as we're committing breakage anyways. But YMMV!